lib/msal-common/src/error/InteractionRequiredAuthError.ts

```diff
@@ -54,7 +54,7 @@ export class InteractionRequiredAuthError extends AuthError {
      * @param errorString 
      * @param subError 
      */
-    static isInteractionRequiredError(errorCode?: string, errorString?: string, subError?: string) : boolean {
+    static isInteractionRequiredError(errorCode?: string, errorString?: string, subError?: string): boolean {
```

Comment:
No real changes in this file

Reply:
Thanks for pointing this out, it helps reviewers.